src/basic/btrfs-util.c

```diff
@@ -45,7 +46,6 @@
 #include "fileio.h"
 #include "io-util.h"
 #include "macro.h"
-#include "missing.h"
```

Comment:
Ah, right, sorry.  Will fix that, unless you don't want the patch anyway, and in that case I won't :)